apps/src/lab2/views/components/Instructions/ForTeachersOnly.tsx

```diff
@@ -1,15 +1,18 @@
-import {useCodebridgeContext} from '@codebridge/codebridgeContext';
 import React from 'react';
 
-import PredictSolution from '@cdo/apps/lab2/views/components/Instructions/PredictSolution';
+import {LevelProperties} from '@cdo/apps/lab2/types';
 import TeacherOnlyMarkdown from '@cdo/apps/templates/instructions/TeacherOnlyMarkdown';
 
-const ForTeachersOnly: React.FunctionComponent = () => {
-  const {levelProperties} = useCodebridgeContext();
+import PredictSolution from './PredictSolution';
+
+const ForTeachersOnly: React.FC<{
```

Comment:
This just moves the existing `ForTeachersOnly` component to the shared lab2 components directory and takes in dependencies as props rather than from codebridge context.